### PR TITLE
coalib/bearlib/languages/documentation/DocumentationComment.py: Refac…

### PR DESCRIPTION
**Pull request by sanketnavin**
…tor line to line number

Renamed the parameter line to line number in the class MalformedComment

Fixes https://github.com/coala/coala/issues/4644

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [x] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [x] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: coalib/bearlib/languages/documentation/DocumentationComment.py [hunk 1]
```diff
@@ -297,17 +297,17 @@ class MalformedComment:
...
-        error has occured.
+        the error: a message explaining the behaviour and a line number where
+        the error has occured.
 
         :param message:
             Contains the message about the error.
-        :param line:
-            Contains the current line of the docstring where the error has
-            occured.
+        :param line_number:
```
Comment (khanchi97) [inline coalib/bearlib/languages/documentation/DocumentationComment.py]: you have to change `line` to `line number` in description only

**Comment by sanketnavin** [comment]
@khanchi97 Alright I will make the changes accordingly

**Comment by SanketDG** [comment]
ask for help in chat if you are stuck with this @sanketnavin

**Comment by stellargo** [comment]
@gitmate-bot mark wip https://github.com/coala/coala/pull/4717

**Comment by stellargo** [comment]
@gitmate-bot Assign wip

**Comment by nvzard** [comment]
This issue(https://github.com/coala/coala/issues/4644) has already been solved and the PR(https://github.com/coala/coala/pull/4758) has been merged.

**Pull request closed without merging**